doc/project/rfc_process.md

```diff
@@ -22,17 +22,22 @@ This document outlines the basic structure of the RFC process, with the expectat
    * If requested, the TC could help to nominate a small group to shepherd the RFC.
 * If the RFC potentially contains information that could be certification-sensitive (guidance to be shared), send a note to security@opentitan.org first for feedback.
 * The RFC is shared publicly by filing a GitHub issue and tagging with the `RFC:Proposal` label.
-* Once the author is happy that the RFC is complete, they submit it to the Technical Committee for review by adding the Github label `TC Review`.
-* The Technical Committee will consider active RFCs in each meeting (those that have been marked ready for at least a week).
+* Once the author is happy that the RFC is complete, they submit it to the Technical Committee by converting the label to `RFC:TC Review`.
+* The Technical Committee will consider active RFCs in each meeting (those that have been marked with `RFC:TC Review` for at least a week).
   If an RFC saw large changes in the week it has been "ready" the TC may postpone judgement in order to allow more comment.
   They will decide whether to:
    * **Accept**: Indicate they are happy for this proposal to be implemented.
      May be conditional on certain changes.
-   * **Reject**: Decided against the proposal at this time
+     The TC will change the label from `RFC:TC Review` to `RFC:Approved` in this outcome.
+   * **Reject**: Decided against the proposal at this time.
+     The label `RFC:TC Review` will be removed by the TC.
    * **Postpone**: The proposal seemed to have the right level of detail and form, but the project won't be making a decision on it at this time (e.g. refers to work that is scheduled too far in the future).
+     The TC will revert the label from `RFC:TC Review` back to `RFC:Proposal`.
```

Comment:
```suggestion
     The TC will revert the label from `RFC:TC Review` back to `RFC:Proposal`.
     In the case where the TC has not had sufficient time to complete review during a meeting, the label `RFC:TC Review` will remain in place until the review has completed.
```

Reply:
wdyt @felixmiller ?

Reply:
Thank you Scott, yes, I think this makes it perfectly clear